docs/examples/echo_server.md

```diff
@@ -86,6 +86,15 @@ In this walkthrough, you'll
 
 1.  Deploy all the echoserver resources (namespace, service, deployment)
 
+    If you're deploying to Fargate, then add the Fargate profile.
+
+    ```bash
+    eksctl create fargateprofile \
+    --cluster <cluster-name> \
+    --name <profile-name> \
+    --namespace echoserver
+    ```
```

Comment:
If not create:
```
$ kubectl describe pod echoserver-dc496b9-lqdcg -n echoserver
Events:

  Warning  FailedScheduling  2m11s  default-scheduler  0/4 nodes are available: 4 node(s) had untolerated taint {eks.amazonaws.com/compute-type : fargate}. preemption: 0/4 nodes are available: 4 Preemption is not helpful for scheduling.
```

Reply:
@wafuwafu13, sorry I added this comment long ago, but just realized I didn't public the comment.

Reply:
Fargate profile is not the only setting for aws-load-balancer-controller, so deleted https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/3322/commits/27d86d6970c555a1d47bf4226505d06d4a95a06a